controllers/topicsController.js

```diff
@@ -65,3 +67,13 @@ exports.postComment = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchArticle = (req, res, next) =>{
+  const {article_id} = req.params
+  const {inc_votes} = req.body
+  Promise.all([getArticleById(article_id), patchArticleById(article_id, inc_votes)])
+  .then((resolvedPromises)=>{
+    res.status(201).send(resolvedPromises[1])
```

Comment:
200 - need to send back and object with the update value + all the other stuff for that article.